test/mobile/test_lite_script_module.py

```diff
@@ -379,8 +379,13 @@ def forward(self, x, y, w):
         loaded = self.getScriptExportImportCopy(ft)
         _, lineno = inspect.getsourcelines(FooTest3)
 
-        with self.assertRaisesRegex(RuntimeError, 'test_lite_script_module.py\", line {}'.format(lineno + 3)):
+        try:
             loaded(torch.rand(3, 4), torch.rand(3, 4), torch.rand(30, 40))
+        except RuntimeError as e:
+            error_message = f"{e}"
+        self.assertTrue('test_lite_script_module.py\", line {}'.format(lineno + 3) in error_message)
```

Comment:
This will fail with a confusing undefined variable error if the module doesn't raise a RuntimeError.